recipes/whylogs/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "whylogs" %}
+{% set version = "1.0.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/whylogs-{{ version }}.tar.gz
+  sha256: cf4068d632e8fc16c01435740fe3336ded20357cde872030b2f2121199ef8fd8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
```

```suggestion
    - python >=3.7
    - poetry-core>=1.0.0
```
Ref: https://github.com/whylabs/whylogs/blob/c1e88184a2e96533659f32f5285ab79aa5ec76ea/python/pyproject.toml#L108